app/models/manageiq/providers/redhat/infra_manager/refresh/parse/strategies/api4.rb

```diff
@@ -1,4 +1,41 @@
 module ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Strategies
   class Api4 < ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Parser
+    def self.cluster_inv_to_hashes(inv)
```

Comment:
Can we use `clusters` instead of `inv`? Would make it easier to read, in my opinion.